libraries/chain/include/eosio/chain/exceptions.hpp

```diff
@@ -390,6 +390,8 @@ namespace eosio { namespace chain {
                                     3080010, "Read-only transaction eos-vm-oc compile permanent failure" )
       FC_DECLARE_DERIVED_EXCEPTION( interrupt_exception, resource_exhausted_exception,
                                     3080011, "Transaction interrupted by signal" )
+      FC_DECLARE_DERIVED_EXCEPTION( interrupt_oc_exception, resource_exhausted_exception,
```

Comment:
Not a big deal, but `interrupt_exception` and `interrupt_oc_exception` are too similar. Maybe `interrupt_by_signal` and `interrupt_by_oc_compile_exception`?

No need to change.